usaspending_api/submissions/management/commands/load_submission.py

```diff
@@ -184,12 +167,16 @@ def handle(self, *args, **options):
             account_balances = None
             try:
                 account_balances = AppropriationAccountBalances.objects.get(tas_rendering_label=row['tas'])
```

Comment:
Yeah, my bad. I meant to convert this to looking by tas_id like earlier in the code. I will fix this.